utilities/blob_db/blob_log_format.cc

```diff
@@ -0,0 +1,252 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
+#ifndef ROCKSDB_LITE
+
+#include "utilities/blob_db/blob_log_format.h"
+#include "util/coding.h"
+#include "util/crc32c.h"
+
+namespace rocksdb {
+namespace blobstorage {
+
+const uint32_t kMagicNumber = 2395959;
+const size_t kBlockSize = 32768;
+
+BlobLogHeader::BlobLogHeader() : magic_number_(kMagicNumber) {}
+
+BlobLogHeader& BlobLogHeader::operator=(BlobLogHeader&& in) noexcept {
+  if (this != &in) {
+    magic_number_ = in.magic_number_;
+    ttl_guess_ = std::move(in.ttl_guess_);
+    ts_guess_ = std::move(in.ts_guess_);
```

Comment:
ttl_guess_ and ts_guess_ consists of primitive integers. The default implementation for primitive types move is simply a copy without any zeroing out on the source part. Thus this move does not really make any difference unless you zero out the source yourself.